### DISCUSSION TITLE
Why is yourls calling all shortened links?

### DISCUSSION THREAD

**Discussion by McFcologne** [YOURLS/YOURLS#3726] [General]
We do log access to the service which is reached by the shortened link. Our logs are full of access from yourls with User-Agent name and IP of our yourls instance:

User-Agent: yourls v1.9.2 +http://yourls.org/ (running on https://ourcompany.link)"/

I cannot see any technical requirement to do so. How can we stop this?

**Comment by dgw** [comment]
It took me a while to figure out what you're asking, but I think you mean that YOURLS is loading the _long URL_ when it is shortened? (At first I thought you described YOURLS accessing itself.)

YOURLS loads the link being shortened to fetch the page title. This functionality can be short-circuited by a simple plugin such as that shown in https://github.com/YOURLS/YOURLS/issues/1966#issuecomment-148905749

**Reply by McFcologne** [reply]
Thanks! But fom my point of view, this behavior is wrong. When you do statistics on the target link, you flood your logs with this waste logs. Yourls should instead force setting a title by the shortening API instead if there is really a demand for this.

**Reply by dgw** [reply]
It is not "wrong" for YOURLS to populate its title field automatically. For most users, this is a convenience so their short URLs are easier to differentiate in the admin list.

Your particular setup simply needs to disable title fetching if you really don't want it, and/or configure your statistics report on the target side to exclude visits with a YOURLS user agent. The web server on your target side could be configured to skip logging YOURLS visits, too ([e.g. for Apache](https://serverfault.com/a/631327/13440)).
